namespaces/live.cloud-platform.service.justice.gov.uk/grc-preprod/resources/rds.tf

```diff
@@ -0,0 +1,58 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+
+module "dps_rds" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.16.12"
+  vpc_name               = var.vpc_name
+  team_name              = var.team_name
+  business-unit          = var.business_unit
+  application            = var.application
+  is-production          = var.is_production
+  namespace              = var.namespace
+  db_engine_version      = "11"
+  environment-name       = var.environment
+  infrastructure-support = var.infrastructure_support
+
+  rds_family = "postgres11"
```

Comment:
Can you specify `db_instance_class = "db.t3.small"` here please, as the default (`db.t2.small`) is previous generation